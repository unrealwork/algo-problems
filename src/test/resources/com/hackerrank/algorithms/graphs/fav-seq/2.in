2
3
68 7 18
10
67 46 11 99 48 69 57 19 45 95
