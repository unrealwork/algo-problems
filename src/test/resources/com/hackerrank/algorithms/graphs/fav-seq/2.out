67 46 11 68 7 18 99 48 69 57 19 45 95

